docs/GUIDE_TO_KOGITO_DEVS.md

```diff
@@ -40,7 +40,7 @@ To generate the jar artifacts, you can just run the following command on each re
 mvn clean package -DskipTests #(For skipping tests) -DskipTestsIT #(For verification)
 ```
 
-This will deploy your jar's in the target directory. For example, for jobs-service the jar would present at `/path/to/kogito-apps/jobs-service/target/jobs-service-8.0.0-SNAPSHOT-runner.jar`
+This will deploy your jar's in the target directory. For example, for jobs-service the jar would present at `/path/to/kogito-apps/jobs-service/target/jobs-service-{VERSION}-runner.jar`
```

Comment:
```suggestion
This will deploy your jar's in the target directory. For example, for jobs-service the jar would present at `/path/to/kogito-apps/jobs-service/jobs-service-common/target/jobs-service-common-{VERSION}-runner.jar`
```

Reply:
done